misc/plot-bamstats

```diff
@@ -1204,6 +1207,119 @@ sub plot_indel_cycles
     plot($$args{gp});
 }
 
+sub plot_read_len_hist
+{
+    my ($opts) = @_;
+
+    my @rl = get_values($opts, 'RL');
+    if ( !@rl ) { return; }
+
+    # TODO: Add modified version for paired end reads.
+    my $is_paired = get_value($opts,'SN','reads paired:');
```

Comment:
Understood that this is a TODO, and OK with that, but maybe we should comment out the `$is_paired` assignment too as it's unused (for now).

Reply:
Done